.eslintrc

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": ["airbnb", "prettier"],
+  "extends": ["airbnb"],
```

Comment:
what's wrong with prettier?

Reply:
`prettier` blocks the `eslint.autoFixOnSave` 🤦‍♂ 
and eslint + aribnb config can do the same thing.

I also changed `npm run format` to `npm run lint` for git hook.